apps/console/src/app.tsx

```diff
@@ -320,6 +327,12 @@ export const App: FunctionComponent<Record<string, never>> = (): ReactElement =>
                                                 <>
                                                     <Helmet>
                                                         <title>{ appTitle }</title>
+                                                        { window?.themeHash && window?.publicPath ?
```

Comment:
Lets reformat the ternary.
https://github.com/wso2/identity-apps/blob/master/docs/DEVELOPER.md#ternary-expressions